### ISSUE TITLE
`minSpareRows` creates a conflict when we ask for the `dataset.length`.

### ISSUE THREAD

**Issue by AMBudnik** [handsontable/angular-handsontable#161]
### Description

`minSpareRows` creates a conflict when we ask for the `dataset.length`.

`To be defined by a developer
`
### Steps to reproduce

1. Just open the console to see the `ExpressionChangedAfterItHasBeenCheckedError` being called.

### Demo

https://jsfiddle.net/f6sL54n1/

### Your environment
* Angular version: `7.2.15`
* Handsontable version: `7.0.0`
* Wrapper version: `5.1.0`
* Browser Name and version: Chrome 76
* Operating System: Windows 10

**Comment by eTallang** [comment]
Any news on this issue? We're still forced to have all tests for the componennt using `minSpareRows` skipped due to this issue.

**Comment by AMBudnik** [comment]
True. Still happens with the new Handsontable 7.2.2 and the latest wrapper version https://jsfiddle.net/1a7ktn4z/

**Comment by swistach** [comment]
This bug should be resolved by Handsontable v8.0.0 where `minSpareRows` is not a part of the dataset. It's not strictly related to the wrapper.

**Comment by swistach** [comment]
Hi @eTallang 
We merged wrapper for Angular into our main repository, [handsontable/handsontable](https://github.com/handsontable/handsontable). Now library and wrapper are available in monorepo.
We also transfer all issues into one common [issue board](https://github.com/handsontable/handsontable/issues), where you can follow work progress and request issues.
